actions/messagebird-send-sms/sms.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+require_once __DIR__.'/../../autoload.php';
+
+use Exo\Toolkit\Runner;
+
+$run = Runner::run(function ($request) {
+    $input = $request['input'];
+
+    $originator = $input['originator'];
+    $recipients = $input['recipients'];
+    $text = $input['text'];
+    $key = $input['key'];
+
+    $MessageBird = new \MessageBird\Client($key);
```

Comment:
linkorb/php-standards: PSR-12: Declare Statements, Namespace, and Import Statements http://criterion.linkorb.com/l/Z-4hsRytRM2z0bDzenwC7Q